Source/WebKit/WebProcess/GPU/media/AudioTrackPrivateRemote.h

```diff
@@ -53,17 +53,17 @@ class AudioTrackPrivateRemote final : public WebCore::AudioTrackPrivate {
 
     using AudioTrackKind = WebCore::AudioTrackPrivate::Kind;
     AudioTrackKind kind() const final { return m_kind; }
-    AtomString label() const final { return m_label; }
-    AtomString language() const final { return m_language; }
+    AtomString label() const final { return AtomString { m_label }; }
```

Comment:
Is there a benefit to returning AtomString here instead of a String? Same question applies elsewhere. Given that the data member is now a String, I'm a little worried about perf if we just keep atomizing the string every time we call the getter. 2 benefits from using an AtomString are:
1. Memory saving thanks to deduplication but that's really only useful for storage. Here the data member is a String.
2. Faster comparison between AtomStrings than between Strings.

I am not familiar with this code. Just though I would make sure you thought of the benefits / drawbacks.

Reply:
No, there's no benefit to return an AtomString here; but every other implementations are using AtomStrings , including the DOM API.

Changing the base class definition is going to open a pandora box; and there's *lots* of stuff using AtomString

Reply:
Also, the string we originally got from the GPU process was a String, made into an AtomString before reaching us. 

In any case, that string needs to be sent to both the HTMLMediaElement's Track, and the SourceBuffer's Track so a copy would have been made one way or another along the way.

Ideally, we would have thread-safe ref-counted string.